.github/workflows/handle.yml

```diff
@@ -127,17 +128,16 @@ jobs:
       # Request will not be handled when tool.yaml file does not exist in the
       # case of a pull_request_review event, instead it prints out a notice.
       - name: 'Handle Tool Request'
+        if: '${{ hashFiles(''tool.yaml'') != '''' }}'
+        id: 'handle_tool'
         env:
           TOOL_FILE_PATH: '${{ github.workspace }}/tool.yaml'
         run: |
-          if [ -f "tool.yaml" ]; then
-            aod tool do -path ${{ env.TOOL_FILE_PATH }}
-          else
-            echo "::notice title=Tool Request Handle::Skip because tool.yaml is not found"
-          fi
+          touch ${{ env.TOOL_ERROR_FILENAME }}
+          aod tool do -path ${{ env.TOOL_FILE_PATH }} 2> ${{ env.TOOL_ERROR_FILENAME }}
 
       - name: 'IAM Request Comment'
-        if: '${{ always()}}'
+        if: '${{ always() && hashFiles(''iam.yaml'') != '''' }}'
```

Comment:
Help me understand - is `always()` useless here?

Reply:
No, always means no matter previous steps failed or succeed or skipped, without this, default is all previous steps must succeed.